opentracing/src/main/java/opentracing/Tracer.java

```diff
@@ -25,20 +25,19 @@
    *
    * <p>Example:
    * <pre>{@code
-   * Tracer tracer = ...
-   * Span feed = tracer.startTrace("GetFeed", null);
-   * Span http = tracer.startTrace("HandleHTTPRequest", feed)
-   *                   .setTag("user_agent", req.UserAgent)
-   *                   .setTag("lucky_number", 42);
-   * }</pre>
-   */
-  Span startSpan(String operationName, /* @Nullable */ Span parent);
+    Tracer tracer = ...
 
-  /**
-   * Same as {@link #startSpan(String, Span)},
-   * but allows to specify a past timestamp in microseconds when the Span was created.
+    Span feed = tracer.startSpan("GetFeed")
+                      .start();
+
+    Span http = tracer.startSpan("HandleHTTPRequest")
+                      .withParent(feed)
+                      .withTag("user_agent", req.UserAgent)
+                      .withTag("lucky_number", 42)
+                      .start();
+    }</pre>
    */
-  Span startSpan(String operationName, long microseconds, /* @Nullable */ Span parent);
+  SpanBuilder startSpan(String operationName);
```

Comment:
buildSpan(..) naturally reads better. I was only worried about how the Specification needs to be met.

I will change to buildSpan(..)